droptools-example/src/main/java/com/bendb/example/ExampleApp.java

```diff
@@ -32,13 +34,26 @@ public FlywayFactory getFlywayFactory(ExampleConfig configuration) {
         bootstrap.addBundle(new JooqBundle<ExampleConfig>() {
             @Override
             public DataSourceFactory getDataSourceFactory(ExampleConfig configuration) {
-                return configuration.dataSourceFactory();
+                return null;
             }
 
             @Override
             public JooqFactory getJooqFactory(ExampleConfig configuration) {
                 return configuration.jooq();
             }
+
+            @Override
+            public DataSourceFactory getDefaultDataSourceFactory(ExampleConfig configuration) {
+                return configuration.dataSourceFactoryMaster();
+            }
+
+            @Override
+            public Map<String,DataSourceFactory> getDataSourceFactories(ExampleConfig configuration) {
+                final Map<String,DataSourceFactory> dataSourceFactoryMap = new HashMap<>();
+                dataSourceFactoryMap.put("master", configuration.dataSourceFactoryMaster());
```

Comment:
We can force people using this bundle to implement an enum with the hardcoded strings.